idaes/core/util/misc.py

```diff
@@ -29,6 +29,22 @@
 _log = idaeslog.getLogger(__name__)
 
 
+# Author: Andrew Lee
+def get_default_solver():
+    """
+    Tries to set-up the default solver for testing, and returns None if not
+    available
+    """
+    if pyo.SolverFactory('ipopt').available(exception_flag=False):
+        solver = pyo.SolverFactory('ipopt')
```

Comment:
It doesn't really mater, but we'll probably want to standardize on "ipopt-idaes" here.  The default settings will probably be the same as plain ipopt forever, but just for future consistency and to set a good example.